ga4gh/backend.py

```diff
@@ -10,6 +10,8 @@
 
 import ga4gh.protocol as protocol
 
+import guppy
```

Comment:
Ah, I think I see how this works now. We shouldn't import guppy here as this creates an unnecessary dependency on the backend.